Readme overhaul (#2988)
Ryujinx/Ryujinx: README.md

<h1 align="center">
  <br>
  <a href="https://ryujinx.org/"><img src="https://i.imgur.com/WcCj6Rt.png" alt="Ryujinx" width="150"></a>
  <br>
  <b>Ryujinx</b>
  <br>

>   <sub><sup><b>(REE-YOU-JI-NX)</b></sup></sub>

Reviewer: I don't think including the spelling here is necessary. Does anyone even spell it like that?

  <br>

</h1>

<p align="center">
       Ryujinx is an open-source Nintendo Switch emulator created by gdkchan and is written in C#.
       This emulator aims at providing excellent accuracy and performance, while also providing a user-friendly interface, and consistent builds.
       It was written from scratch and development started around September of 2017. Ryujinx is available on GitHub under the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/LICENSE.txt" target="_blank">MIT license</a>. <br /> 

</p>
<p align="center">
    <a href="https://ci.appveyor.com/project/gdkchan/ryujinx?branch=master">
        <img src="https://ci.appveyor.com/api/projects/status/ssg4jwu6ve3k594s/branch/master?svg=true"
            alt="">
    </a>
    <a href="https://discord.com/invite/VkQYXAZ">
        <img src="https://img.shields.io/discord/410208534861447168?color=5865F2&label=Ryujinx&logo=discord&logoColor=white"
            alt="Discord">
    </a>
    <br>
    <br>
    <img src="https://raw.githubusercontent.com/Ryujinx/Ryujinx-Website/master/static/public/shell_fullsize.png">
</p>

<h5 align="center">
  
</h5>

## Compatibility

> As of September 2021, Ryujinx has been tested on nearly 3,400 titles: ~3,000 boot past menus and into gameplay, with approximately 2,400 of those being considered playable.

Reviewer 2: As of January 2022, Ryujinx has been tested on approximately 3,500 titles; over 3,200 boot past menus and into gameplay, with roughly 2,500 of those being considered playable.

You can check out the compatibility list [here](https://github.com/Ryujinx/Ryujinx-Games-List/issues). Anyone is free to submit an updated test on an existing game entry; simply follow the new issue template and testing guidelines, and post as a reply to the applicable game issue.

Don't hesitate to open a new issue if a game isn't already on there!
                  
## Usage

To run this emulator, we recommend that your PC has at least 8GB of RAM; less than this amount can result in unpredictable behavior, crashes or unacceptable performance.

See our [Setup & Configuration Guide](https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide) on how to set up the emulator.

> For our Local Wireless and LAN builds See our [Multiplayer: Local Play/Local Wireless Guide

Reviewer 3: ```suggestion
For our Local Wireless and LAN builds, see our [Multiplayer: Local Play/Local Wireless Guide
```

](https://github.com/Ryujinx/Ryujinx/wiki/Multiplayer-(LDN-Local-Wireless)-Guide)

## Latest build

These builds are compiled automatically for each commit on the master branch. While we strive to ensure optimal stability and performance prior to pushing an update, our automated builds **may be unstable or completely broken.**

> If you want to see what's changed in an update you can visit our [Changelog](https://github.com/Ryujinx/Ryujinx/wiki/Changelog).

Reviewer 3: ```suggestion
If you want to see what's changed in an update, you can visit our [Changelog](https://github.com/Ryujinx/Ryujinx/wiki/Changelog).
```

The latest automatic build for Windows, macOS, and Linux can be found on the [Official Website](https://ryujinx.org/download).

## Building

If you wish to build the emulator yourself, follow these steps:

### Step 1
Install the X64 version of [.NET 6.0 (or higher) SDK](https://dotnet.microsoft.com/download/dotnet/6.0).

### Step 2

> Either use `git clone https://github.com/Ryujinx/Ryujinx` to clone the repository or use the `Code>Download zip` button to get the files

Reviewer 4: ```suggestion
Either use `git clone https://github.com/Ryujinx/Ryujinx` to clone the repository or use the `Download zip` button to get the files
```

Author: Imo I don’t like removing `code` as not everyone would know/see where the download button is, so I’m leaving that change there to make it more streamlined for newer users.

> Either use `git clone https://github.com/Ryujinx/Ryujinx` to clone the repository or use the `Code>Download zip` button to get the files.

Reviewer 4: ```suggestion
Either use `git clone https://github.com/Ryujinx/Ryujinx` on the command line to clone the repository or use Code --> Download zip button to get the files.
```

### Step 3

> To build Ryujinx open the Command prompt in the project directory. You can quickly access it by holding shift in the file explorer(while inside the Ryujinx directory) then right clicking and selecting ``, then type the following command:

Reviewer 3: ```suggestion
To build Ryujinx, open a command prompt inside the project directory. You can quickly access it on Windows by holding shift in File Explorer, then right clicking and selecting `Open command window here`. Then type the following command:
```

## Contact

If you have contributions, suggestions, need emulator support or just want to get in touch with the team, join our [Discord server](https://discord.com/invite/Ryujinx). You may also review our [FAQ](https://github.com/Ryujinx/Ryujinx/wiki/Frequently-Asked-Questions).

## Donations
                  
If you'd like to support the project financially, Ryujinx has an active Patreon campaign.

<a href="https://www.patreon.com/ryujinx">
    <img src="https://images.squarespace-cdn.com/content/v1/560c1d39e4b0b4fae0c9cf2a/1567548955044-WVD994WZP76EWF15T0L3/Patreon+Button.png?format=500w" width="150">
</a>

All the developers working on the project do so on their free time, but the project has several expenses:
* Hackable Nintendo Switch consoles to reverse-engineer the hardware
* Additional computer hardware for testing purposes (e.g. GPUs to diagnose graphical bugs, etc.) 
* Licenses for various software development tools (e.g. Jetbrains, LDN servers, IDA)

> * Web hosting and infrastructure maintinance

Reviewer 2: Typo.
"maintenance" is the proper spelling.

Author: Agh don’t know how I let that one slip I’ll fix it asap

   The emulator has settings for enabling or disabling some logging, remapping controllers, and more. You can configure all of them through the graphical interface or manually through the config file, `Config.json`, found in the user folder which can be accessed by clicking `Open Ryujinx Folder` under the File menu in the GUI.

## Contact

> If you are having problems launching homebrew or a particular game marked status-playable or status-ingame in our compatibility list, or if you need help with setting up Ryujinx we recommend you ask questions in the #support channel of our [Discord server](https://discord.gg/Ryujinx) or take a look at our <a href="https://github.com/Ryujinx/Ryujinx/wiki/Frequently-Asked-Questions" target="_blank">FAQ.</a></i><br />  If you have contributions, have suggestions, or just want to get in touch with the team join our [Discord server](https://discord.gg/Ryujinx).

Reviewer 4: ```suggestion
If you are having problems launching homebrew or a particular game marked status-playable or status-ingame in our compatibility list, or if you need help with setting up Ryujinx we recommend you ask questions in the #support channel of our [Discord server](https://discord.gg/Ryujinx) or take a look at our <a href="https://github.com/Ryujinx/Ryujinx/wiki/Frequently-Asked-Questions" target="_blank">FAQ.</a></i><br />  If you have contributions, have suggestions, or just want to get in touch with the team, join our [Discord server](https://discord.gg/Ryujinx).
```

## Donations
                  
If you'd like to support us financially, please take a look at our Patreon

<a href="https://www.patreon.com/ryujinx">
    <img src="https://images.squarespace-cdn.com/content/v1/560c1d39e4b0b4fae0c9cf2a/1567548955044-WVD994WZP76EWF15T0L3/Patreon+Button.png?format=500w" width="150">
</a>

All the developers working on the project do so on their free time, but the project has several expenses:

> * Hackable Nintendo Switch consoles to reverse-engineer the hardware and its components

Reviewer: `and its components` is redundant.

> * Hardware to test and fix any bugs certain hardware may have

Reviewer: Hardware was already listed.

Reviewer: I guess you're talking about PC parts rather than Switch hardware here? If so you should clarify.

> * Additional hardware to test and fix any bugs certain vendors (AMD, Nvidia, and Intel) may have 

Reviewer 4: ```suggestion
* Additional hardware to test (eg. GPUS to fix graphical bugs, other accessories to add support for, etc.) 
```

> * Software licenses to help developers use whatever they are comfortable with

Reviewer: `allow` rather than `help`.

> * Software licenses to allow developers use whatever they are comfortable with

Reviewer 4: Ditto here

Reviewer 4: Change to `Software licences (Visual Studio, etc)` etc, up to you.

Author: Imo not needed.

> * Software licenses to allow developers use whatever they are comfortable with

Reviewer 2: Development on this emulator takes place during contributors' free time. However, there are some inherent costs:
* Hackable Nintendo Switch consoles to reverse-engineer the hardware
* Additional computer hardware for testing purposes (e.g. GPUs to diagnose graphical bugs, etc.)
* Licenses for various software development tools

> * Web hosting and infrastructure 

Reviewer 4: ```suggestion
* Web hosting and infrastructure maintaining
```

> All the funds acquired through our Patreon is considered a donation to support our work, and we will not do patron only releases. We may post exclusive progress reports, or early access to tools that can be used with the emulator.

Reviewer: `are` rather than `is`?

Reviewer: > or early access to tools that can be used with the emulator

Remove this as we are not doing that right now.

Reviewer: I'd also remove
 > , and we will not do patron only releases

> All the funds acquired through our Patreon are considered a donation to support our work. Patrons get 1 week of exclusive early access to our progress reports and access to developer interviews.

Reviewer 4: ```suggestion
All the funds acquired through our Patreon are considered a donation to support our work. Patrons will get 1 week of exclusive early access to our progress reports and access to developer interviews.
```

> All the funds acquired through our Patreon are considered a donation to support our work. Patrons will get 1 week of exclusive early access to our progress reports and access to developer interviews.

Reviewer 2: * Web hosting and infrastructure maintenance

All funds received through Patreon are considered a donation to support the project. Patrons receive early access to progress reports and exclusive access to developer interviews.

## License

> This software is licensed under the terms of the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/LICENSE.txt" target="_blank">MIT license.</a></i><br /> 

Author: Ditto

> The Ryujinx.Audio project is licensed under the terms of the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/Ryujinx.Audio/LICENSE.txt

Reviewer 4: ```suggestion
The Ryujinx.Audio project is licensed under the terms of <a href="https://github.com/Ryujinx/Ryujinx/blob/master/Ryujinx.Audio/LICENSE.txt
```

Author: Imo removing that is unnecessary

" target="_blank">LGPLv3 license.</a></i><br />
This project makes use of code authored by the libvpx project, licensed under BSD and the ffmpeg project, licensed under LGPLv3.
See [LICENSE.txt](LICENSE.txt) and [THIRDPARTY.md](Ryujinx/THIRDPARTY.md) for more details.
## Credits

> - [LibHac](https://github.com/Thealexbarney/LibHac) is used for our file-system 

Reviewer: ```suggestion
- [LibHac](https://github.com/Thealexbarney/LibHac) is used for our file-system implementation.
```
